java/sage/Wizard.java

```diff
@@ -3156,6 +3177,11 @@ public SeriesInfo addSeriesInfo(int legacySeriesID, int showcardID, String title
         series.seriesImages = Pooler.EMPTY_INT_ARRAY;
       else
         series.seriesImages = seriesImages.clone();
+      if (imageURLs == null || imageURLs.length == 0)
+        series.imageURLs = Pooler.EMPTY_2D_BYTE_ARRAY;
+      else
+        series.imageURLs = imageURLs;
+      series.imageURLs = (imageURLs == null) ? Pooler.EMPTY_2D_BYTE_ARRAY : imageURLs.clone();
```

Comment:
You're setting series.imageURLs twice here. Just remove this last line and then add a .clone() on the above line (yeah, I know it's not fully consistent about cloning array arguments on all the object creations, I used to be paranoid but got less paranoid as time went on)